src/main/java/com/alkemy/ong/mapper/SlideMapper.java

```diff
@@ -9,19 +10,28 @@
 @Component
 public class SlideMapper {
 
-  public SlideResponse map(Slide slide) {
+  public SlideResponse map(Slide slide, String operation) {
     SlideResponse slideResponse = new SlideResponse();
-    slideResponse.setImageUrl(slide.getImageUrl());
-    slideResponse.setText(slide.getText());
-    slideResponse.setOrder(slide.getOrder());
-    return slideResponse;
+    if (operation.equals(SlideOperation.ALL.name())) {
+      slideResponse.setImageUrl(slide.getImageUrl());
+      slideResponse.setText(slide.getText());
+      slideResponse.setOrder(slide.getOrder());
+      return slideResponse;
+    } else if (operation.equals(SlideOperation.IMAGEORDER.name())) {
+      slideResponse.setImageUrl(slide.getImageUrl());
+      slideResponse.setOrder(slide.getOrder());
+      return slideResponse;
+    } else {
+      return null;
```

Comment:
BUENAS PRACTICAS: Los métodos nunca deberían devolver null. Si tengo un método que devuelve null, es una clara señal de que algo esta mal en el diseño del mismo. En este caso, debería utilizar un switch y el el default lanzar un `new UnsupportedOperationException(MessageFormat.format("Slide operation: {0} is unsupported", operation))`